### PR TITLE
[126] - Sprint4 fixing

### PR DESCRIPTION
**Pull request by otash8110**
### Where

[NL-126 Sprint4 fixing](https://jjbcapital.atlassian.net/browse/NL-126)

### What

Fix functionality implemented in sprint 4

### How

Change browser tab icon and title, remove starting price limit, messsage when adding existing images and when adding existing files

### Affected Area

Auctionify.API.ClientApp
Auctionify.Infrastructure
Auctionify.Application.Features

### PR Checklist

_Build_

- [x] Build Succeeded

_Unit Tests_

- [x] 46 Tests Passed, 0 Ignored, 0 Failed

### CODE REVIEW

**Review by khasanrashidov — APPROVED** [review]
Looks Good To Me! Every issue that was addressed during the review was fixed. Approved!

File: src/Auctionify.API/ClientApp/src/app/components/seller/create-lot/create-lot.component.ts [hunk 1]
```diff
@@ -360,7 +359,6 @@ export class CreateLotComponent implements OnInit {
         if (this.lotForm.controls.startingPrice.value !== null)
             this.lotForm.controls.startingPrice.addValidators([
                 Validators.min(1),
-                Validators.max(9999),
```
Comment (TsvirkoNikita) [inline src/Auctionify.API/ClientApp/src/app/components/seller/create-lot/create-lot.component.ts:363]: It's not necessary to completely remove the upper limit for the price, it's enough to raise it to a large value, I think up to a trillion would be enough

**Review by TsvirkoNikita — CHANGES_REQUESTED** [review]

File: src/Auctionify.API/ClientApp/src/app/components/seller/create-lot/create-lot.component.ts (hunk 1 above)
Comment (khasanrashidov) [reply src/Auctionify.API/ClientApp/src/app/components/seller/create-lot/create-lot.component.ts:363]: Fixed, maximums is trillion, the validation message is fixed as well ("maximum starting price is 1000000000000")

File: src/Auctionify.API/ClientApp/src/app/components/seller/create-lot/pop-ups/currency/currency.component.ts [hunk 2]
```diff
@@ -41,7 +41,6 @@ export class CurrencyPopUpComponent {
         if (this.startingPriceFormGroup.controls.startingPrice.value !== null)
             this.startingPriceFormGroup.controls.startingPrice.addValidators([
                 Validators.min(1),
-                Validators.max(9999),
```
Comment (TsvirkoNikita) [inline src/Auctionify.API/ClientApp/src/app/components/seller/create-lot/pop-ups/currency/currency.component.ts:44]: Update validation here too

**Review by TsvirkoNikita — CHANGES_REQUESTED** [review]

File: src/Auctionify.API/ClientApp/src/app/components/seller/create-lot/pop-ups/currency/currency.component.ts (hunk 2 above)
Comment (khasanrashidov) [reply src/Auctionify.API/ClientApp/src/app/components/seller/create-lot/pop-ups/currency/currency.component.ts:44]: corrected validation for max in currency component as well

**Review by TsvirkoNikita — APPROVED** [review]

**Review by khasanrashidov — APPROVED** [review]

**Pull request merged**